source/partials/community/_events.slim

```diff
@@ -56,8 +56,8 @@
 .Panel.Panel--alternative
   .HorizontalGrid.HorizontalGrid--limited
     #bragajs.HorizontalGrid-cell
-      .GridCell
-        .GridCell-content
+      .GridCell.u-flexGrow.u-defaultMargin
+        .GridCell-content.StickToTheBottom.u-defaultThenLargeMargin
```

Comment:
not to be too picky, but isn't the `The` part of the name unnecessary? `StickToBottom` would work just the same